src/iris/api.py

```diff
@@ -574,9 +577,15 @@ def ts_to_sql_datetime(ts):
 uuid4hex = re.compile('[0-9a-f]{32}\Z', re.I)
 
 
-def stream_incidents_with_context(cursor):
+def stream_incidents_with_context(cursor, title=False):
     for row in cursor:
         row['context'] = ujson.loads(row['context'])
+        if title:
+            title_variable_name = row['title_variable_name']
+            if title_variable_name:
+                row['title'] = row['context'][title_variable_name]
```

Comment:
Let's do a `.get(title_variable_name)` just in case

Reply:
done